CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 0.6.0
+
+ * Update dependency to angular: ^4.0.0.
```

Comment:
Super fun, I know – would you collapse the changelog into one entry for v0.6.0?

Reply:
Oh yeah, I was wondering about that. I'll go through and smash them together.

Reply:
👍